app/_src/policies/meshcircuitbreaker.md

```diff
@@ -0,0 +1,303 @@
+outlier detection basis. For outlier detection to open circuit breaker you can configure what we call "detectors".
+This policy provides 5 different types of detectors, and they are triggered on some deviations in the upstream service
+behavior. All detectors could coexist on the same outbound interface.
+
+Once one of the detectors has been triggered the corresponding data plane proxy is ejected from the set of the load
+balancer for a period equal to [baseEjectionTime](#outlier-detection-configuration). Every further ejection of the same data plane
+proxy will further extend the [baseEjectionTime](#outlier-detection-configuration) multiplied by the number of ejections: for example
+the 4th ejection will be lasting for a period of time of `4 * baseEjectionTime`.
+
+This policy provides **passive** checks.
+If you want to configure **active** checks, please utilize the [MeshHealthCheck](/docs/{{ page.version }}/policies/meshhealthcheckcz)
+policy.
+Data plane proxies with **passive** checks won't explicitly send requests to other data plane proxies to determine if
+target proxies are healthy or not.
+
+## TargetRef support matrix
+
+| TargetRef type    | top level | to  | from |
+|-------------------|-----------|-----|------|
+| Mesh              | ✅         | ✅   | ✅    |
+| MeshSubset        | ✅         | ❌   | ❌    |
+| MeshService       | ✅         | ✅   | ❌    |
+| MeshServiceSubset | ✅         | ❌   | ❌    |
+| MeshGatewayRoute  | ❌         | ❌   | ❌    |
+
+To learn more about the information in this table, see the [matching docs](/docs/{{ page.version }}/policies/targetref).
+
+### Examples
+
+#### Basic circuit breaker for outbound traffic from web, to backend service
+
+{% tabs usage useUrlFragment=false %}
+{% tab usage Kubernetes %}
+
+```yaml
+apiVersion: kuma.io/v1alpha1
+kind: MeshCircuitBreaker
+metadata:
+  name: web-to-backend-circuit-breaker
+  namespace: {{site.mesh_namespace}}
+spec:
+  targetRef:
+    kind: MeshService
+    name: web
+  to:
+    - targetRef:
+        kind: MeshService
+        name: backend
+      default:
+        connectionLimits:
+          maxConnections: 2
+          maxPendingRequests: 8
+          maxRetries: 2
+          maxRequests: 2
+```
+
+We will apply the configuration with `kubectl apply -f [..]`.
+{% endtab %}
+
+{% tab usage Universal %}
+
+```yaml
+type: MeshCircuitBreaker
+name: web-to-backend-circuit-breaker
+mesh: default
+spec:
+  targetRef:
+    kind: MeshService
+    name: web
+  to:
+    - targetRef:
+        kind: MeshService
+        name: backend
+      default:
+        connectionLimits:
+          maxConnections: 2
+          maxPendingRequests: 8
+          maxRetries: 2
+          maxRequests: 2
+```
+
+We will apply the configuration with `kumactl apply -f [..]` or via the [HTTP API](/docs/{{ page.version }}/reference/http-api).
+{% endtab %}
+{% endtabs %}
+
+#### Outlier detection for inbound traffic to backend service
```

Comment:
I know it's not strictly related to docs but how does outlier detection work on an inbound?

Reply:
Well it would act more like a inbound rate limiter, but since it passed MADR then I'm not going to protest.